cg/store/filters/status_flow_cell_filters.py

```diff
@@ -1,17 +1,22 @@
 from enum import Enum
-from typing import Optional, List, Callable
+from typing import Callable, List, Optional
 
 from sqlalchemy.orm import Query
 
-from cg.store.models import Flowcell, FamilySample, Family
+from cg.store.models import Family, FamilySample, Flowcell
 
 
 def filter_flow_cells_by_case(case: Family, flow_cells: Query, **kwargs) -> Query:
     """Return flow cells by case id."""
     return flow_cells.filter(FamilySample.family == case)
 
 
-def get_flow_cell_by_name(flow_cells: Query, flow_cell_name: str, **kwargs) -> Query:
+def filter_flow_cells_by_has_backup(flow_cells: Query, **kwargs) -> Query:
```

Comment:
Not sure if it would be useful to also add a function to filter flow cells that do NOT have a backup.